crates/pop-cli/src/commands/call/chain.rs

```diff
@@ -238,7 +244,7 @@ impl CallChainCommand {
 		spinner.start("Signing and submitting the extrinsic and then waiting for finalization, please be patient...");
 		let call_data_bytes =
 			decode_call_data(call_data).map_err(|err| anyhow!("{}", format!("{err:?}")))?;
-		let result = sign_and_submit_extrinsic_with_call_data(client, call_data_bytes, suri)
+		let result = sign_and_submit_extrinsic(client, url, CallData::new(call_data_bytes), suri)
 			.await
 			.map_err(|err| anyhow!("{}", format!("{err:?}")))?;
 
```

Comment:
If I execute the call directly with the call_data the output is not well formatted (Try with You can test it with` pop call chain --call 0x040000d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d025a6202`)
 Is a very simple change :
```
spinner.stop(format!("Extrinsic submitted successfully with hash: {:?}", result));
```
with:
```
spinner.stop(result);
```

Reply:
Thank you for noting this!